[bugfix] worker is not respected when loadbalance:false (#51041)
elastic/beats: docs/reference/winlogbeat/logstash-output.md

### `loadbalance` [loadbalance]

When `loadbalance: true` is set, Winlogbeat connects to all configured hosts and sends data through all connections in parallel. If a connection fails, data is sent to the remaining hosts until it can be reestablished. Data will still be sent as long as Winlogbeat can connect to at least one of its configured hosts.

> When `loadbalance: false` is set, Winlogbeat sends data to a single host at a time. Winlogbeat connects to the first configured host and sends all data to that host until the connection fails, then fails over to the next configured host in order. Data will still be sent as long as Winlogbeat can connect to at least one of its configured hosts. To rotate through the list of configured hosts over time, use this option in conjunction with the `ttl` setting to close the connection at the configured interval and reconnect to the next host in the list.

Reviewer: ```suggestion
Set `loadbalance: false` to send data to one host at a time. Winlogbeat connects to the first configured host and sends all data to that host. If the connection fails, Winlogbeat fails over to the next configured host. As long as Winlogbeat can connect to at least one configured host, it continues to send data.

To rotate through the list of configured hosts over time, use this option in conjunction with the `ttl` setting to close the connection at the configured interval and reconnect to the next host in the list.
```
